src/components/profile/Profile.js

```diff
@@ -147,11 +148,35 @@ export function Profile({ match }) {
         })()
     }, []);
 
+    useEffect(() => {
+        let id = Mixpanel.get_distinct_id()
+        Mixpanel.identify(id)
+        Mixpanel.people.set_once({create_date: new Date().toString(),})
+        Mixpanel.people.set({
+            relogin_date: new Date().toString(),
+            enabled_2FA: account.has2fa,
+            [accountId]: formatNEAR(account.balance.total) })
+        Mixpanel.alias(accountId)
+    },[])
+
+    useEffect(() => {
+        if (userRecoveryMethods) {
+            let id = Mixpanel.get_distinct_id()
+            Mixpanel.identify(id)
+            let methods = userRecoveryMethods.map(method => method.kind)
+            Mixpanel.people.set({recovery_method: methods})
+        }
+    },[userRecoveryMethods])
+
     useEffect(()=> {
         if(twoFactor){
             let id = Mixpanel.get_distinct_id()
             Mixpanel.identify(id)
-            Mixpanel.people.set({create_2FA_at: twoFactor.createdA, enable_2FA_kind:twoFactor.kind, enabled_2FA: true })
+            Mixpanel.people.set({
+                create_2FA_at: twoFactor.createdAt, 
```

Comment:
Do we need to track `twoFactor` info in `Profile.js` since it's already tracked in `Routing.js`?

Reply:
I delete the tracking in the routing.js and move them here